R/rrq.R

```diff
@@ -33,12 +32,30 @@ hipercow_rrq_controller <- function(..., set_as_default = TRUE, driver = NULL,
                                     queue_id = NULL, root = NULL) {
   root <- hipercow_root(root)
   call <- rlang::current_env()
+
```

Comment:
Perhaps `loadNamespace` with `versionCheck` here to make sure we have `0.7.20` or later

Reply:
Used `compareVersion` instead for nicer error messages.